lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -64,6 +64,7 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
     typeId = 'AttachFileCloudWidgetComponent';
     rootNodeId = ALIAS_USER_FOLDER;
     selectedNode: Node;
+    lastSentToViewers: Node;
```

Comment:
hmm I am not sure if the name is correct, can this attach widget be used without viewer? if yes, we should not tightly couple it with some outer component